Update helium-wallet-app.mdx (#1156)
helium/docs: docs/wallets/helium-wallet-app.mdx

The Helium Wallet app is the trusted way to manage your Helium tokens and identity.

> Some notable features from v2.x onwards

Reviewer: ```suggestion
Some notable features:
```
I think we keep this as-is.

- Multiple Account Support
- Ledger Balance
- Address Book
- Verify your Helium Identity

> - Lock-up tokens for voting power and stakin gincome

Reviewer: ```suggestion
- Lock-up tokens for voting power and staking rewards
```

- Voting on governance
- Multiple Recipient Payment Support

> - Claimiing tokens from hotspots to your wallet

Reviewer: ```suggestion
- Claiming tokens from Hotspots to your wallet
```

> - Transfer NFTs representing hotspot ownership

Reviewer: ```suggestion
- Transfer NFTs representing Hotspot ownership
```
Hotspots always proper noun

  src={useBaseUrl('img/wallets/helium-wallet-app/address-book-banner.png')}
/>

> This feature allows you to save names/nicknames and related Helium or Solana allet addresses for 

Reviewer: ```suggestion
This feature allows you to save names/nicknames and related wallet addresses for 
```
Prefer to not draw to much contrast between Helium/Solana accounts since they're one in the same.
